%YAML:1.0

lidar_to_imu_rotation: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data: [ 7.533745000000e-03, -9.999714000000e-01, -6.166020000000e-04,
          1.480249000000e-02, 7.280733000000e-04, -9.998902000000e-01,
          9.998621000000e-01, 7.523790000000e-03, 1.480755000000e-02]

lidar_to_imu_translation: !!opencv-matrix
  rows: 3
  cols: 1
  dt: d
  data: [ -4.069766000000e-03, -7.631618000000e-02, -2.717806000000e-01 ]